.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
+.xml
+.idea
```

Comment:
Add an empty line on end of every file, coding standards

Reply:
Done